server/src/tasks/schema.js

```diff
@@ -104,7 +92,14 @@ const taskResolvers = {
   }
 }
 
-function publish(actionType, data) {
+function publish(actionType, data, pushClient) {
```

Comment:
It will be good to get that into Subscription/Notification file.

Reply:
Yeah. this will be completely gone soon. I am going to completely refactor this stuff so we have one synchronisation package in voyager-server.